neuralpp/symbolic/sympy_expression.py

```diff
@@ -36,7 +36,6 @@ def _infer_sympy_object_type(sympy_object: sympy.Basic, type_dict: Dict[sympy.Ba
             return fractions.Fraction
         case sympy.logic.boolalg.BooleanAtom():
             return bool
-        # TODO: delete me
```

Comment:
Line #39 and #40 cannot be deleted because otherwise we cannot recognize an internal class of sympy.Piecewise (`sympy.functions.elementary.piecewise.ExprCondPair`)